app/classifier/classifier.py

```diff
@@ -23,75 +22,68 @@ class Classifier:
     def __init__(self, learning_mode=False):
         """Constructor. Loads learning model from file."""
         if not learning_mode:
-            file_with_model = open(MODEL_FILE, 'rb')
+
+            try:
```

Comment:
http://stackoverflow.com/questions/82831/how-to-check-whether-a-file-exists-using-python#82852